### PR TITLE
Create logos for Gazorator and initial readme

### PR DESCRIPTION
**Pull request by khalidabuhakmeh**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
+# Grazorator
```
Comment (bjorkstromm) [inline readme.md]: nit: Gazorator

File: readme.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+![gazorator logo](./assets/grazorator-horizontal-logo.png)
+
+Grazorator is a `dotnet` tool that empowers developers to generate XML based outputs using the power of the Razor View Engine.
```
Comment (bjorkstromm) [inline readme.md]: Gazorator is not a tool, it's a library which allows rendering Razor templates outside of ASP.NET Core.

**Review by bjorkstromm — COMMENTED** [review]
Thanks again @khalidabuhakmeh for the nice logos snd README! Unfortunately the name contains a typo.

File: readme.md (hunk 1 above)
Comment (khalidabuhakmeh) [reply readme.md]:
```suggestion
# Gazorator
```

File: readme.md (hunk 2 above)
Comment (khalidabuhakmeh) [reply readme.md]:
```suggestion
Grazorator is a `dotnet` library that empowers developers to generate XML based outputs using the power of the Razor View Engine.
```

**Comment by khalidabuhakmeh** [comment]
Where dafuq did I get `grazorator` from?! My brain is broken. Let me fix this PR up.

**Comment by gep13** [comment]
@bjorkstromm was there a reason that this wasn't merged in?

These logos look amazing!

**Comment by bjorkstromm** [comment]
@gep13 just me missing that @khalidabuhakmeh had fixed the PR feedback 😄 

Thanks again @khalidabuhakmeh for the awesome logos and README!

**Pull request merged**